python/chronos/src/bigdl/chronos/forecaster/base_forecaster.py

```diff
@@ -242,7 +243,21 @@ def fit(self, data, validation_data=None, epochs=1, batch_size=32, validation_mo
                batch_size setted in `data`.if the forecaster is distributed, the batch_size will be
                evenly distributed to all workers.
         :param validation_mode: Operation mode while having 'validation_data'. Defaults to 'output'.
-        :return: A dict that records the average validation loss of each epoch.
+               The validation_mode includes the following types:
+
+               | 1. output:
+               | If you choose 'output' for validation_mode, it will return a dict that records the
+               | average validation loss of each epoch.
+               |
+               | 2. earlystop:
+               | Monitor the val_loss and stop training when it stops improving.
+
+        :param earlystop_patience: Number of checks with no improvement after which training will
+               be stopped. It takes effect when 'validation_mode' is 'earlystop'. Under the default
+               configuration, one check happens after every training epoch. However, the frequency
```

Comment:
Our users can not set `check_val_every_n_epoch` and `val_check_interval` at least for now. Please remove the comment after "however"